Project.toml

```diff
@@ -23,10 +24,12 @@ StatsBase = "0.24, 0.32, 0.33"
 julia = "1"
 
 [extras]
+DelayDiffEq = "bcd4f6db-9728-5f36-b5f7-82caef46ccdb"
```

Comment:
I've just checked the tests, but I didn't see `DelayDiffEq` used somewhere. Is it necessary? If you need a MackieGlass timeseries, we now have it uploaded (for the Nickhwade PR)